packages/angular/package.json

```diff
@@ -33,37 +34,34 @@
   "license": "Apache-2.0",
   "author": "IBM",
   "peerDependencies": {
-    "@angular/common": "^9.0.0 || ^10.0.0 || ^11.0.0",
-    "@angular/core": "^9.0.0 || ^10.0.0 || ^11.0.0",
-    "@angular/forms": "^9.0.0 || ^10.0.0 || ^11.0.0",
+
     "carbon-components": "^10.0.0",
     "carbon-components-angular": "^4.0.0",
-    "rxjs": "^6.0.0",
-    "zone.js": "^0.8.26 || ^0.9.0 || ^0.10.0"
+    "rxjs": "^6.0.0"
```

Comment:
`peerDependencies` should stay peer dependencies. Angular dependencies should be extended to include 12 (and maintain backwards compatibility).

rxjs should probably stay here and be removed from regular dependencies.